src/pages/difficulty.tsx

```diff
@@ -85,6 +33,24 @@ const DifficultyPage = (): React.ReactElement => {
                     </li>
                 ))}
             </ul>
+
+            <h2>Tool Kit DCs</h2>
+            <div>
+                {DC_TOOLS.map(tool => (
+                    <div key={tool.name}>
+                        <h2>{tool.name}</h2>
+                        <ul>
+                            {tool.dcs.map((dc, index) => (
+                                <li key={index}>
+                                    <span>
+                                        {dc.text} {dc.difficulty}
+                                    </span>
```

Comment:
I would wrap each of the text values in their own spans in case you want to style them separately.